app/src/main/java/com/main/mealplanner/MainActivity.kt

```diff
@@ -32,4 +34,10 @@ class MainActivity : AppCompatActivity() {
                 .commitNow()
     }
 
+    fun goBack(view: View) {
+        val intent = Intent(this@MainActivity, MainActivity::class.java)
+        startActivity(intent)
+        finish()
+    }
+
```

Comment:
https://developer.android.com/training/basics/firstapp/starting-activity

I notice that if I clicked on MoreInfo, I did not see a way back the the pervious page / Main fragment. So I put this in place in order to be able to go back, other wise I'd have exit the app and go back in. Now This is a starting place to do so and you change this how ever you like.

Reply:
Does the back button work?  

There's an Android design principle that you shouldn't duplicate system functions, like the back button.  Less clutter that way; no need for two back buttons.

Reply:
Yes, ran the project on my machine and worked